### ISSUE TITLE
confd is too silent in the logfile

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#491]
Originally reported of Google Code with ID 435.

```
when running eg. "gnt-group list"

confd responds to the query, but nothing is added to the logfile unless it's in debugging mode. While it's ok not to have deep debugging information having an entry that says "I got a query of this type", and perhaps "I responded without errors" would help.

```
Originally added on 2013-04-24 06:43:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-24 06:45:29 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I'll take care of this.
```
Originally added on 2013-04-26 12:10:43 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 3e0c2a24a53823b36a2cd5f8a79393fcd9c998a1
Author: Klaus Aehlig <aehlig@google.com>
Date:   Mon Apr 29 14:08:17 2013 +0200

    Log at info level the success of a luxi query
    
    So far, any successful query was only logged at debug level, thus leaving
    no traces in the logs, unless debugging was enabled. Therefore, we add
    a notes about successful completion at info level. Also, repeat a failed
    request at WARN level, so that, even if debugging disabled, the request
    is available together with the error message in the logs.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2013-04-29 15:12:03 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
